### PR TITLE
Add deeplink information

### PR DESCRIPTION
**Pull request by muthuswamyopn**
Added a para to state that on payment methods that require opening the external app (e.g. mobile banking app) to authorize the transaction, one must set the return_uri to a deeplink or applink to be able to open the merchant app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -458,7 +458,9 @@ style.xml
 ```
 
 ## Authorizing Payment
-Some payment methods require the customer to authorize the payment via an authorization URL. This includes the [3-D Secure verification](https://www.omise.co/fraud-protection#3-d-secure), [Internet Banking payment](https://www.omise.co/offsite-payment), [Alipay](https://www.omise.co/alipay), etc. Omise Android SDK provides a built in class to handle the authorization.
+Some payment methods require the customer to authorize the payment via an authorization URL. This includes the [3-D Secure verification](https://www.omise.co/fraud-protection#3-d-secure), [Internet Banking payment](https://www.omise.co/offsite-payment), [Alipay](https://www.omise.co/alipay), etc. Opn Payments Android SDK provides a built in class to handle the authorization.
```
Comment (danfowler) [inline README.md:461]: @muthuswamyopn can you take the opportunity to replace the omise.co links with their equivalent on docs.opn.ooo (making sure the new link still works)

**Review by nuxzero — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (muthuswamyopn) [reply README.md:461]: I will make the changes right away today, Dan. Apologies for missing this comment.

Comment (muthuswamyopn) [reply README.md]: Updated links.

Should we change email to support@opn.ooo?

Comment (muthuswamyopn) [reply README.md]: Updated support email

**Review by danfowler — APPROVED** [review]

**Pull request merged**